hack/vela-templates/definitions/expose.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: core.oam.dev/v1beta1
+kind: TraitDefinition
+metadata:
+  annotations:
+    definition.oam.dev/description: "expose port to enable web traffic for your service."
+  name: expose
+  namespace: {{.Values.systemDefinitionNamespace}}
+spec:
+  appliesToWorkloads:
+    - webservice
```

```suggestion
    - deployments.apps
```
please change to workloadDefinition name, please refer to https://github.com/oam-dev/kubevela/issues/1635

---

yes, i'll fix